tests/server_test_cases/admin_statistics_tests.py

```diff
@@ -68,3 +67,45 @@ def test_note_counter(self):
         # one person's record and num_notes of notes are created
         notes = doc.cssselect_one('#haiti-notes')
         assert notes.text == str(num_notes)
+        # test unspecified status note counter
+        unspecified_status = doc.cssselect_one('#haiti-unspecified')
+        assert unspecified_status.text == str(num_notes)
+
+    def test_is_note_author_counter(self):
+        """Test if the is_note_author_counter increment when choose
+        I want to input my own information in the create form"""
+        doc = self.go('/japan/create?given_name=_test_given_name&'
+                      'family_name=_test_family_name&role=provide')
+        form = doc.cssselect_one('form')
+        self.s.submit(form,
+                      own_info='yes',  # status == 'is_note_author'
+                      given_name='_test_given_name',
+                      family_name='_test_family_name',
+                      text='_note_text')
+        doc = self.go_as_admin('/global/admin/statistics')
+        assert 'japan' in doc.text
+        is_note_author_counter = doc.cssselect_one('#japan-is_note_author')
+        assert is_note_author_counter.text == '1'
+
+    def test_status_counter(self):
+        """Test of counter increment based on the given status_name"""
+        def increment_counter(status_name, amount):
```

Comment:
Maybe better to name it increment_counter_and_assert because it asserts as well as increments counter.